share/ert/forward-models/shell/careful_copy_file

```diff
@@ -0,0 +1,9 @@
+STDOUT      careful_copy_file.stdout
```

Comment:
I suggest we remove all lines from these files but the `EXECUTABLE`. Everything else is either sensibly defaulted or not relevant for a simulation job :+1: